README.md

```diff
@@ -24,6 +24,24 @@ import 'details-dialog-element'
 </details>
 ```
 
+## Events
+
+### `details-dialog:will-close`
+
+A `details-dialog:will-close` event is fired when a request to close the dialog
+is made either by pressing escape, clicking the close button, clicking outside
```

Comment:
Should replace "clicking outside the dialog" with "clicking on `<summary>`. The "outside of dialog" overlay layer is a Primer helper which expands `<summary>` to fullscreen, but it doesn't come with `<details-dialog>`.